environments/base/default-config.libsonnet

```diff
@@ -125,17 +128,20 @@
   query: {
     image: defaultConfig.thanosImage,
     version: defaultConfig.thanosVersion,
+    replicas: 1,
   },
 
   queryFrontend: {
     image: defaultConfig.thanosImage,
     version: defaultConfig.thanosVersion,
+    replicas: 1,
   },
 
   api: {
     local apiConfig = self,
-    version: 'master-2020-09-04-v0.1.1-135-g7c621cf',
+    version: 'master-2020-09-08-v0.1.1-142-g61a908f',
```

Comment:
Could you elaborate why this bump is relevant for replica configurability?

Reply:
@periklis Thanks for your comments. I should have another PR to do this upgrade. Actually, it is related with https://github.com/observatorium/deployments/pull/334